packages/mobile/src/transactions/send.ts

```diff
@@ -158,9 +158,7 @@ export function* sendTransactionPromises(
   }
 
   const feeCurrencyAddress =
-    feeCurrency === CURRENCY_ENUM.DOLLAR
-      ? yield call(getCurrencyAddress, CURRENCY_ENUM.DOLLAR)
-      : undefined // Pass undefined to use CELO to pay for gas.
+    feeCurrency === Currency.Dollar ? yield call(getCurrencyAddress, Currency.Dollar) : undefined // Pass undefined to use CELO to pay for gas.
```

Comment:
It seems we should handle Euro too.

Reply:
Also done in the Send cEUR PR, didn't want to add too much more stuff in this one